iset.mm

```diff
@@ -3,17 +3,17 @@
 $( Metamath is a formal language and associated computer program for
    archiving, verifying, and studying mathematical proofs, created by Norman
    Dwight Megill (1950--2021).  For more information, visit
-   http://us.metamath.org and
+   https://us.metamath.org and
    https://github.com/metamath/set.mm, and feel free to ask questions at
    http://groups.google.com/group/metamath. $)
 
-$( New users may want to read http://us.metamath.org/ileuni/conventions.html to
-   understand the label naming conventions used in iset.mm.  See also "help
+$( New users may want to read https://us.metamath.org/ileuni/conventions.html
+   to understand the label naming conventions used in iset.mm.  See also "help
    verify markup" in the Metamath program for markup conventions. $)
 
-$( To break this file into smaller modules, in the Metamath program type "read
-   iset.mm" followed by "write source iset.mm /split"; to recombine, omit
-   "/split". $)
+$( To break this file into smaller modules, in the Metamath program type
+   "MM> READ iset.mm" followed by "MM> WRITE SOURCE iset.mm / SPLI".  To
```

Comment:
```suggestion
   "MM> READ iset.mm" followed by "MM> WRITE SOURCE iset.mm / SPLIT".  To
```

Reply:
I cannot apply it. Maybe you can ?

Reply:
(I keep seeing "This diff has recently been updated. Please refresh and try again." but refreshing has no effect but returning to the initial stage; this seems to be a bug in github.)

Reply:
I get the same message,

Reply:
I did a "manual" PR, could not find any other way.

Reply:
Thanks, and sorry for the inconvenience !